src/dataexplorer/dataexplorer-application/src/components/QuestionnaireResponse/QuestionnaireResponseProcessing/SchemaView/SchemaView.tsx

```diff
@@ -84,12 +84,110 @@ const SchemaView = (props: SchemaViewProps) => {
+                                                                        display.id.split(
+                                                                            '.',
+                                                                        )[1] ===
+                                                                            '101' && (
+                                                                            <Col
+                                                                                span={
+                                                                                    18
+                                                                                }
+                                                                                order={
+                                                                                    1
+                                                                                }
+                                                                                key={
+                                                                                    display.id
+                                                                                }
+                                                                                className="section-description"
+                                                                            >
+                                                                                {
+                                                                                    display
+                                                                                        .questions
+                                                                                        .questions
+                                                                                        .text
+                                                                                }
+                                                                            </Col>
+                                                                        ),
+                                                                )}
+                                                                <Col
+                                                                    span={6}
+                                                                    order={2}
+                                                                    className="section-description"
+                                                                >
+                                                                    {!sectionDescription.includes(
+                                                                        section.id.split(
+                                                                            '.',
+                                                                        )[0],
```

Comment:
I don't think you need to split here, sine section.id wont contain '.'. Replacing this with `section.id` should be the same.